src/main/java/eu/dissco/backend/repository/ElasticSearchRepository.java

```diff
@@ -204,10 +204,21 @@ public Pair<Long, Map<String, Map<String, Long>>> getAggregation(MappingTerms ma
         .trackTotalHits(t -> t.enabled(Boolean.TRUE))
         .aggregations(mappingTerm.getName(),
             AggregationBuilders.terms().field(mappingTerm.getFullName()).build()._toAggregation())
+        .size(0)
         .build();
     var aggregation = client.search(aggregationRequest, ObjectNode.class);
     var totalRecords = aggregation.hits().total().value();
     var aggregationResult = collectResult(aggregation.aggregations());
     return Pair.of(totalRecords, aggregationResult);
   }
+
+  public Map<String, Map<String, Long>> searchTermValue(String name, String field, String value) throws IOException {
```

Comment:
What's the difference between name and field

Reply:
So the name would be `country` and the field would be `digitalSpecimen.ods:attributes.dwc:country.keyword`

Reply:
We set it here so that the bucketName is equal to the original name we got from the frontend instead of the full elastic name